bokeh/plotting/tests/test_figure.py

```diff
@@ -235,6 +237,19 @@ def test_title_kwarg_no_warning(recwarn):
     assert len(recwarn) == 0
 
 
+def test_figure_title_should_accept_title():
+    title = Title(text='Great Title')
+    plot = plt.figure(title=title)
+    l1 = plot.line([1, 2, 3], [1, 2, 3])
```

Comment:
the `l1` is causing flake errors. either add the #noqa or don't assign the renderer to anything